tasks/copy_to_glacier/test/unit_tests/test_handler.py

```diff
@@ -205,6 +201,49 @@ def test_task_empty_granules_list(self):
         s3_cli.head_object.assert_not_called()
         s3_cli.copy.assert_not_called()
 
+    @patch.dict(os.environ, {"ORCA_DEFAULT_BUCKET": ""}, clear=True)
+    def test_task_invalid_environment_variable(self):
+        """
+        Checks to make sure an unset or empty environment variable throws an error
+        """
+        collection_name = uuid.uuid4().__str__()
+        collection_version = uuid.uuid4().__str__()
+        destination_bucket_name = os.environ['ORCA_DEFAULT_BUCKET']
+        source_bucket_name = uuid.uuid4().__str__()
+        collection_url_path = uuid.uuid4().__str__()
+        content_type = uuid.uuid4().__str__()
+        source_bucket_names = [file['bucket'] for file in self.event_granules['granules'][0]['files']]
+        source_keys = [file['filepath'] for file in self.event_granules['granules'][0]['files']]
+
+        # todo: use 'side_effect' to verify args. It is safer, as current method does not deep-copy args
+        boto3.client = Mock()
+        s3_cli = boto3.client('s3')
+        s3_cli.copy = Mock(return_value=None)
+        s3_cli.head_object = Mock(return_value={'ContentType': content_type})
+
+        event = {
+            'input': copy.deepcopy(self.event_granules),
+            'config': {
+                CONFIG_COLLECTION_KEY: {
+                    COLLECTION_NAME_KEY: collection_name,
+                    COLLECTION_VERSION_KEY: collection_version,
+                },
+                CONFIG_BUCKETS_KEY: {
+                    'bad_bucket': {
+                        'name': 'not_glacier'
+                    }
+                }
```

Comment:
Either remove this, or add in the 'glacier' bucket if you want to verify that a missing default now blows everything up.

Reply:
Removed the buckets object.